concepts/modularization/modularization_overview.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Modularization in Kyma
+---
+
+With Kyma’s modular approach, you can install exactly the Kyma components you want, making the installation very lightweight and adjusted to your business needs. Kyma uses three system components to support the modular approach: The Module Manager, the Lifecycle Manager, and the Runtime Watcher.
+
+## Kyma’s Modular Approach
+<!-- diagram still WIP! -->
+![](assets/modular_approach.svg)
+<!-- alternative draft: -->
+![](assets/modular_approach2.svg)
+
+## 1. Kyma Module
+
+A Kyma module contains all information required to install and run the associated components in a Kyma runtime:
+
+* One or more renderable Kubernetes manifests of the module, based on Kustomize and/or Helm
+* A default configuration for module chart resource installation
+* Optionally, further layers that can enhance, or be required by, the operator of a module
+
+All these assets are bundled into a single container image using the [OCI image specification](https://github.com/opencontainers/image-spec).
+
+> **TIP:** Learn how to build a module(ADD_LINK).
```

Comment:
Can you add the link, or is it not available for now?

Reply:
I don't think it exists already :/

Reply:
Currently, it'S this: https://github.com/kyma-project/lifecycle-manager/blob/main/docs/developer/creating-test-environment.md#create-module-operator-and-bundle-module

BUT, there is a bigger change happening which will simplify things massively (see [issue](https://github.com/kyma-project/lifecycle-manager/issues/226))

Reply:
So we don't link it and wait for the simplified version?

Reply:
I suggest you link https://github.com/kyma-project/lifecycle-manager/tree/main/samples/template-operator as its our official reference implementation with step-by-step guides for setting up the operator and building the module. We will change this as well once the CLI is adapted but for this you dont have to adjust the link. There will be a move of this repository in a few weeks probably out of lifecycle-manager for easier access, then we will need to revisit this article

Reply:
```suggestion
> **TIP:** Learn how to [build a module](https://github.com/kyma-project/lifecycle-manager/tree/main/samples/template-operator).
```